frontend/src/old-pages/Clusters/JobSubmitDialog.tsx

```diff
@@ -263,6 +270,23 @@ export default function JobSubmitDialog({
           </FormField>
         </SpaceBetween>
 
+        {
+          isMemBasedSchedulingEnabled && <SpaceBetween direction="vertical" size="xxs" key="mem">
+          <Header variant="h2"
+            description={t("JobSubmitDialog.requiredMemory.description")}>
+            <Trans i18nKey="JobSubmitDialog.requiredMemory.header" />
```

Comment:
Why are you using _Trans_ instead of the standard _useTranslation_ function used above in this case?

Reply:
That's a good point, I had started using `Trans` and then used the function because I was unable to use `Trans` for the description, trying with the function directly.

Reply:
Okay, verified that we can do everything with the function, changed in rev2.